server/status.js

```diff
@@ -0,0 +1,316 @@
+/*
+  Apparently, the new api.export takes care of issues here. No need to attach to global namespace.
+  See http://shiggyenterprises.wordpress.com/2013/09/09/meteor-packages-in-coffeescript-0-6-5/
+
+  We may want to make UserSessions a server collection to take advantage of indices.
+  Will implement if someone has enough online users to warrant it.
+*/
+import { Accounts } from 'meteor/accounts-base';
+import { check, Match } from 'meteor/check';
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { _ } from 'meteor/underscore';
```

Comment:
Do we absolutely need underscore?

Reply:
I removed some places underscore was used, but it will be a bit of work going around and replacing all the other uses. I would say that should be in a new PR.

Reply:
Could be a good issue for new comers. :+1: